games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+VIRTUALX_REQUIRED="test"
+
+inherit java-utils-2 virtualx
+
+DESCRIPTION="A Rom Manager entirely written in Java and released under GPL v3"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip"
+QA_PREBUILT="*"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND="${DEPEND}
+	app-arch/unzip"
+
+src_install() {
+	insinto "/usr/share/${PN}"
+	doins *.jar
+	doins -r lib
+	exeinto "/usr/share/${PN}"
+	sed -e "/dir=.*/s:.*:cd ${EROOT}usr/share/${PN}:" -i *.sh
```

Comment:
why not from `src_prepare()` ?

Reply:
Because I **install** binaries and do not compile

Reply:
does not matter:

1.) Any files changes (sed or patching) have to be done in `src_prepare()`
2.) sed should end up with `|| die`
3.) You can not act on the live fs there, it should be at least `${ED}` but as I said move it to prepare